src/users/users.controller.ts

```diff
@@ -13,11 +13,14 @@ import { UsersService } from './users.service';
 import { CreateUserDto } from './dto/create-user.dto';
 import { UpdateUserDto } from './dto/update-user.dto';
 import { IdValidationPipe } from 'src/common/pipes/id-validation.pipe';
+import { Auth } from 'src/auth/decorators/auth.decorator';
+import { AuthRoles } from 'src/common/enums/roles.enum';
 
 @Controller('users')
 export class UsersController {
   constructor(private readonly usersService: UsersService) {}
 
+  @Auth(AuthRoles.Admin)
```

Comment:
If you want to add quard to every method you can do it in such way:
@Auth(AuthRoles.Admin)
@Controller('users')